eth/gasprice/feehistory.go

```diff
@@ -275,9 +276,9 @@ func (oracle *Oracle) FeeHistory(ctx context.Context, blocks int, lastBlock rpc.
 	for ; blocks > 0; blocks-- {
 		fees := <-results
 		if fees.err != nil {
-			return 0, nil, nil, nil, fees.err
+			return common.Big0, nil, nil, nil, fees.err
 		}
-		i := int(fees.blockNumber - oldestBlock)
+		i := int(fees.blockNumber) - oldestBlock
```

Comment:
I think we can also use the `uint64` for `fees.blockNumber`. And only convert it to `rpc.BlockNumber` when call some backend APIs.